src/database/models/acommodationlikesandunlikes.js

```diff
@@ -0,0 +1,21 @@
+export default(sequelize, DataTypes) => {
+  const AcommodationLikesAndUnlikes = sequelize.define('AcommodationLikesAndUnlikes', {
+    userId: DataTypes.STRING,
+    name: DataTypes.STRING,
+    imageUrl: DataTypes.STRING,
+    accommodationId: DataTypes.STRING,
+  }, {});
+  AcommodationLikesAndUnlikes.associate = (models) => {
+    AcommodationLikesAndUnlikes.belongsTo(models.Users, {
+      foreignKey: 'userId',
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+    });
```

Comment:
@RusimbiP refering to what @blaise82 said, if you have this relationship you don't really need those other user's details you are saving.

Reply:
The user details be helpful on the front-end

Reply:
I did it on purpose

Reply:
I want to be displaying the image and name of the person who liked something by querying the likes DB only. Once and for all.

Reply:
@RusimbiP that's the point of the relationship, if u have the id of the one who liked/unliked the accommodation, you can always get his/her details from the id without having to save the user's details in likes table, cos that is repetition

Reply:
I think in this context we will not need associations. Ditch the other `GET` let us get it once and for all. I think that is more efficient in this context. I don't think duplication of text data in tables is as equally. costly as querying it many times. I'm still learning this relationship thing tho.

@AbdoulNuru @blaise82

Reply:
Feedback Attend to. Thanks, @AbdoulNuru

Reply:
@RusimbiP good job